projects/ngx-diagrams/src/lib/defaults/models/default-label.model.ts

```diff
@@ -1,9 +1,10 @@
-import { LabelModel } from '../../models/label.model';
 import { BehaviorSubject, Observable } from 'rxjs';
+import { LabelModel } from '../../models/label.model';
 
 export class DefaultLabelModel extends LabelModel {
-	private _label: BehaviorSubject<string> = new BehaviorSubject('');
-	label$: Observable<string> = this._label.pipe(this.entityPipe('label'));
+	protected _label = new BehaviorSubject('');
+
+	label$ = this._label.asObservable().pipe(this.entityPipe('label'));
```

Comment:
piping a behaviorSubject turns it into an observable automatically, using asObservable is redundant